src/utils/PermissionUtils.js

```diff
@@ -13,4 +13,12 @@ module.exports = class PermissionUtils {
       return member.roles.filter(r => r.hoist).sort((a, b) => b.position - a.position).first()
     }
   }
+
+  static async isDj (client, user, channel) {
+    const guild = channel.guild
+    const { role } = await client.database.guilds.get(guild.id)
+    if (!role) return
+    const hasRole = role && role.members.has(user.id)
+    return hasRole
```

Comment:
by the way, that would return undefined in any way, since `{ role }` doesn't exist on the schemas: it's `{ djRole }`

Reply:
aaaaaaaaand, lines 20 to 22 can be reduced to `return djRole ? djRole.members.has(user.id) : undefined`